src/main/java/org/sagebionetworks/web/client/SynapseJSNIUtils.java

```diff
@@ -94,6 +94,8 @@ public void getFilePartMd5(
 
   public void loadCss(String url);
 
+  public String[] getSrcPersistentLocalStorageKeys();
```

Comment:
Add a method to convert SRC's persistent local storage keys from a JS array to a Java array in this interface, so that this method can be mocked in AuthenticationControllerImplTest. Happy to adjust if there's a better place or way to handle this case!

Reply:
It doesn't 100% make sense here since you're using JSInterop, not JSNI. But I think it's fine here compared to other alternatives because it's a set of similar utilities. I'm ok with it.